packages/react/src/components/ErrorSummary/ErrorSummary.tsx

```diff
@@ -0,0 +1,56 @@
+import type { HTMLAttributes } from 'react';
+import { createContext, forwardRef, useId, useState } from 'react';
+import cl from 'clsx';
+import { Slot } from '@radix-ui/react-slot';
+
+import type { ListProps } from '../List';
+import { List } from '../List';
+
+import classes from './ErrorSummary.module.css';
+
+type ErrorSummaryContextType = {
+  headingId?: string;
+  setHeadingId: (id: string) => void;
+};
+
+export const ErrorSummaryContext = createContext<ErrorSummaryContextType>({
+  headingId: 'heading',
+  setHeadingId: () => {},
+});
```

Comment:
This is not needed though? New `List` is already handling this functionality for us.

Reply:
We need the id on the div that has `role="alert"`, and since the ID is sent only to `ErrorSummary.Heading`, we need this 😓

Reply:
Doh! true true 🙈